hoodie-client/src/main/java/com/uber/hoodie/table/HoodieTable.java

```diff
@@ -201,22 +202,20 @@ public HoodieTimeline getCommitTimeline() {
   /**
    * Get only the completed (no-inflights) compaction commit timeline
    */
-  public HoodieTimeline getCompletedCompactionCommitTimeline() {
-    return getCompactionCommitTimeline().filterCompletedInstants();
+  public HoodieTimeline getCompletedCompactedCommitTimeline() {
+    return getCompactedCommitTimeline().filterCompletedInstants();
   }
 
 
   /**
    * Get the compacted commit timeline visible for this table
    */
-  public HoodieTimeline getCompactionCommitTimeline() {
+  public HoodieTimeline getCompactedCommitTimeline() {
```

Comment:
Can rename this to `getCommitTimeline()` but having compacted in the name makes things a little clear. I'm open to renaming this.

Reply:
I think we can rename to `getCommitTimeline()` and `getCompletedCommitTimeline()` ( and also cleanup any duplicate methods)

Reply:
The problem is getCompletedCommitTimeline() at the moment assumes we return commit + deltacommit for MOR table which is correct. "Completed" just means filtering out in-flight. We cannot use "completed" and/or "compacted" interchangeably since many places assume completed as filtered only.

Reply:
lets rename this also to `getCommitsTimeline()` consistent with above.. thats the method called here anyways